res/layout/launcher_activity.xml

```diff
@@ -2,6 +2,7 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="fill_parent" android:layout_height="fill_parent" android:orientation="vertical">
   <TextView style="@style/paragraph" android:text="@string/launcher_description"/>
   <Button style="@style/paragraph" android:text="@string/launcher_button_imesettings" android:onClick="launch_imesettings" android:layout_width="wrap_content"/>
+ <Button style="@style/paragraph" android:text="set keyboard" android:onClick="launch_imepicker" android:layout_width="wrap_content"/>
```

Comment:
I love this idea! Would you mind creating a different PR from a different branch to separate the button and the themes ?

Reply:
I honestly didn't even remember that change, but it's okay, the button is to change the keyboard from the app, since it is difficult to put the keyboard in if you don't know but don't worry, I'll do it now